### PR TITLE
Bitcoin Core 26.2

### PR DESCRIPTION
**Pull request by glozow**
final changes PR: https://github.com/bitcoin/bitcoin/pull/30376

### CODE REVIEW

File: _releases/26.2.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- #30085: p2p: detect addnode cjdns peers in GetAddedNodeInfo()
+
+### RPC
+
+- #29869: rpc, bugfix: Enforce maximum value for setmocktime
+- #28554: bugfix: throw an error if an invalid parameter is passed to getnetworkhashps RPC
+- #30094: rpc: move UniValue in blockToJSON
+- #29870: rpc: Reword SighashFromStr error message
+
+### Build
```
Comment (stickies-v) [inline _releases/26.2.md:85]: As per my comment [here](https://github.com/bitcoin/bitcoin/pull/30376#pullrequestreview-2153177017), the release notes are incomplete, this should be rebased on latest `26.x`

Comment (glozow) [reply _releases/26.2.md:85]: Copied incorrectly, even still had "rc1" in the title! :facepalm: fixed now

File: _releases/26.2.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+- Jameson Lopp
+- jonatack
+- laanwj
+- Luke Dashjr
+- MarcoFalke
+- nanlour
+- willcl-ark
+
+As well as to everyone that helped with translations on
+[Transifex](https://www.transifex.com/bitcoin/bitcoin/).
```
[the remark this answers is not in the document]
Comment (glozow) [reply _releases/26.2.md:121]: added, thanks

**Comment by glozow** [comment]
Thanks! I've added magnet link, fixed @katesalazar's comment, and edited the dates to be today.

**Review by stickies-v — APPROVED** [review]
ACK a95e4a6e8645d64187328bb420241cce27f96a48 - all review comments adressed, verified the magnet link

**Comment by achow101** [comment]
ACK a95e4a6e8645d64187328bb420241cce27f96a48

**Review by Sawsqr68 — COMMENTED** [review]
Stfi🦦😁🙋

**Pull request merged**